test/step_output_test.go

```diff
@@ -96,16 +96,17 @@ func TestStepOutput(t *testing.T) {
 	if err != nil {
 		t.Errorf("Error getting Taskrun %q: %v", taskRun.Name, err)
 	}
-	var gotResult *v1beta1.TaskRunResult
+	var gotResult bool
```

Comment:
this not a result anymore, maybe just "got"?
```suggestion
	var got bool
```